src/components/Committees/Elections/Tabs/Lists.jsx

```diff
@@ -73,100 +89,95 @@ const Lists = ({ election }) => {
-          <Grid item xs={12} sm={6} lg={4} key={list.uuid} data-cy="election-list-card">
-            <Box
-              display="flex"
-              alignItems="center"
-              justifyContent="space-between"
-              sx={{
-                bgcolor: 'whiteCorner',
-                px: 1.5,
-                py: 1,
-                borderTopRightRadius: '6px',
-                borderTopLeftRadius: '6px',
-              }}
-            >
-              <Typography sx={{ fontSize: '16px', color: theme => theme.palette.colors.gray[500] }}>
-                Liste {index + 1}
-              </Typography>
-              {list.candidacies.length === 0 && enabledAction && (
-                <IconButton
-                  edge="start"
-                  color="inherit"
-                  onClick={() => deleteList(list.uuid)}
-                  aria-label="delete"
-                  sx={{ ml: 0.25 }}
-                >
-                  <DeleteIcon sx={{ color: theme => theme.palette.form.error.color, fontSize: '20px' }} />
-                </IconButton>
-              )}
-            </Box>
-            <Accordion>
-              <AccordionSummary
-                expandIcon={<ExpandMoreIcon />}
-                aria-controls={`panel-content-${index + 1}`}
-                id={`panel-header-${index + 1}`}
-              >
-                <Box
-                  sx={{ display: 'flex', flex: '1 1 0%', justifyContent: 'space-between', alignItems: 'center' }}
-                  className="space-x-4"
-                >
-                  <Typography>Les candidats</Typography>
-                </Box>
-              </AccordionSummary>
-              <AccordionDetails>
-                <Box>
-                  {list.candidacies.length === 0 && <Typography>{messages.noCandidate}</Typography>}
-                  {list.candidacies.length > 0 &&
-                    list.candidacies.map(candidate => (
-                      <Box display="flex" alignItems="center" my={1.5} key={candidate.uuid}>
-                        <Typography sx={{ color: theme => theme.palette.colors.gray[900] }}>
-                          {candidate.committee_membership.adherent.first_name}{' '}
-                          {candidate.committee_membership.adherent.last_name}
-                        </Typography>
-                        {enabledAction && (
-                          <IconButton
-                            edge="start"
-                            color="inherit"
-                            onClick={() => deleteCandidate(candidate.uuid)}
-                            aria-label="delete"
-                            sx={{ ml: 1 }}
-                          >
-                            <DeleteIcon sx={{ color: theme => theme.palette.form.error.color, fontSize: '20px' }} />
-                          </IconButton>
-                        )}
-                      </Box>
-                    ))}
+      {election.status === 'closed' && results ? (
```

Comment:
peux-tu utiliser la constante stp? 

```suggestion
      {election.status === electionStatus.closed && results ? (
```